Modules/GLO/src/ITSTPCmatchingCheck.cxx

```diff
@@ -0,0 +1,250 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   ITSTPCmatchingCheck.cxx
+/// \author felix.schlepper@cern.ch
+///
+
+#include "GLO/ITSTPCmatchingCheck.h"
+#include "QualityControl/MonitorObject.h"
+#include "QualityControl/Quality.h"
+#include "QualityControl/QcInfoLogger.h"
+
+#include <Rtypes.h>
+#include <TH1.h>
+#include <TEfficiency.h>
+#include <TPaveText.h>
+#include <TArrow.h>
+#include <TText.h>
+#include <TBox.h>
+#include <TLine.h>
+#include <TLegend.h>
+#include <TFile.h>
+#include <TPolyMarker.h>
+
+#include <DataFormatsQualityControl/FlagReasons.h>
+#include "fmt/format.h"
+
+using namespace std;
+using namespace o2::quality_control;
+
+namespace o2::quality_control_modules::glo
+{
+
+void ITSTPCmatchingCheck::configure()
+{
+  if (auto param = mCustomParameters.find("minPt"); param != mCustomParameters.end()) {
```

Comment:
I would suggest to move the parameters initialization in the `startOfActivity()` function, and use the extended parameters to allow defining beam-type dependent thresholds, if needed.